src/main/java/net/dries007/tfc/client/screen/CreateTFCWorldScreen.java

```diff
@@ -43,15 +42,23 @@ private static OptionInstance<Double> pctOption(String caption, double defaultVa
 
     private static OptionInstance<Integer> kmOption(String caption, int min, int max, int defaultValue)
     {
-        return new OptionInstance<>(caption, OptionInstance.cachedConstantTooltip(Component.translatable(caption + ".tooltip")), (text, value) -> Options.genericValueLabel(text, Component.translatable("tfc.settings.km", String.format("%.1f", value / 1000.0))), new OptionInstance.IntRange(min, max), defaultValue, value -> {});
+        return new OptionInstance<>(caption, OptionInstance.cachedConstantTooltip(Component.translatable(caption + ".tooltip")),
+            (text, value) -> Options.genericValueLabel(text, Component.translatable("tfc.settings.km", String.format("%.1f", value / 1000.0))), new OptionInstance.IntRange(min, max), defaultValue, value -> {});
+    }
```

Comment:
Why this diff?

Reply:
Matches the wrapping of the lines above, much easier to read.